forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -707,7 +707,7 @@ private Card setupStaticEffect(Card copied, SpellAbility cause) {
             // needed for ETB lookahead like Bronzehide Lion
             stAb.putParam("AffectedZone", "All");
             SpellAbilityEffect.addForgetOnMovedTrigger(eff, "Battlefield");
-            game.getAction().moveToCommand(eff, cause);
+            eff.getOwner().getZone(ZoneType.Command).add(eff);
```

Comment:
why not use the moveToCommand one?

Reply:
performance
it would lead to here:
https://github.com/Card-Forge/forge/blob/11913085efe87ad7970a2de305fe288e8e61130c/forge-game/src/main/java/forge/game/GameAction.java#L106-L112

and I didn't like that during resolve